landing_page/views.py

```diff
@@ -3,7 +3,7 @@
 
 # Initial landing page view.
 def index(request):
-    return render(request, "landing_page/index.html")
+    return render(request, "landing_page/movie.html")
```

Comment:
you need to make a different view with this render - we still need the index one